### PR TITLE
Ap/feature branch example

### PR DESCRIPTION
**Pull request by anakp07**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
```
Comment (beauttie) [inline .idea/.gitignore:6]: Git should ignore this

File: README.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ Person B should modify content here.
 ## Person C:
 
 Person C should modify content here.
+This is my Ana's changes
```
Comment (beauttie) [inline README.md:16]: great

File: morning-coffee.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+I love to smell of coffee in the morning :)
```
Comment (beauttie) [inline morning-coffee.txt:1]: ☕

**Review by beauttie — CHANGES_REQUESTED** [review]

File: .idea/.gitignore (hunk 1 above)
Comment (anakp07) [reply .idea/.gitignore:6]: I agree.

File: morning-coffee.txt (hunk 3 above)
Comment (anakp07) [reply morning-coffee.txt:1]: energy is almost there now!

**Pull request merged**